packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "d5746045ac740d5f28f238dc55ec95c89d2b7cd9",
-    "ref_origin" : "dcos-mesos-1.0.x-837bb4e"
+    "ref": "fd011fcce61ecf4c4d4cc3cde09a11c1470ff1b6",
```

Comment:
need an explicit ack from the pods and the like that this is the direction we want / need to go for 1.9. To date we've been intentionally keeping pods a bit separate.